azext_iot/_params.py

```diff
@@ -233,11 +234,27 @@ def load_arguments(self, _):
         context.argument('certificate_path',
                          options_list=['--certificate-path', '-cp'],
                          help='The path to the file containing the primary certificate. '
-                         'One of the certificate path is required.')
+                         'If choose to use intermediate certificate, one of the certificate path is required.')
```

Comment:
This description may be more clear 'If attestation with an intermediate certificate is desired then a certificate path is required.'

Similarly for CA:

'If attestation with a CA certificate is desired then a certificate name is required.'

Reply:
or even 'If attestation with an intermediate certificate is desired then a certificate path must be provided.' ........